admissions/views.py

```diff
@@ -0,0 +1,42 @@
+from django.shortcuts import render
+
+from OneApply.constants import UserType
+from admissions.models import HighSchoolApplication
+from register.models import Admin_Staff
+
+
+def index(request):
```

Comment:
A ListView would've been easier in terms of application and functionality. It gives us the ease of pagination, categorizing data, filtering, and extra context as and when we need it (just the way it's done for HighSchool). This is something that we'll need to revisit post this sprint because the admission_staff applications will end up being a lot more than the number of schools for a student to look at, and that'll be a pain to maintain and manage all those functionalities by ourselves when we can simply use a ListView generic view.

Reply:
Maybe adding a TODO to change this view to ListView in the later sprint is something that could be done for now